modules/project-docs/pages/compatibility.adoc

```diff
@@ -62,17 +62,17 @@ The Couchbase Go SDK is tested and supported on the following platforms:
 [discrete]
 === GNU/Linux
 
-* Amazon Linux 2.
-* Red Hat Enterprise Linux 7 & 8;
-* CentOS 7 & 8.
-* Ubuntu (LTS) 16.04 (_Xenial_), 18.04 (_Bionic_), & 20.04 (_Focal_).
-* Debian 9 (_Stretch_) & 10 (_Buster_).
-* Alpine Linux
+* Amazon Linux 2 & AL2023.
+* Red Hat Enterprise Linux 8 & 9;
+* Alma Linux 9, Rocky Linux 9.
```

Comment:
I see Rocky support coming in Server - but I may be mistaken adding it to SDK at this time?

Reply:
I don't think Ubuntu has been dropped.  Less sure about Debian.